src/test/rustdoc-ui/check-fail.stderr

```diff
@@ -33,10 +32,15 @@ LL | | //! ```
    | |_______^
    |
    = help: the code block will either not be tested if not marked as a rust one or the code will be wrapped inside a main function
+note: the lint level is defined here
+  --> $DIR/check-fail.rs:6:9
+   |
+LL | #![deny(rustdoc::all)]
+   |         ^^^^^^^^^^^^
```

Comment:
Not sure why this changed.

Reply:
Want to investigate a bit further before approval?

Reply:
Ah, I just figured it out: This error is from a different (stable) lint than the unstable `rustdoc::missing_doc_code_examples`. Because there are now two `deny`s (one for `rustdoc::all` and one for `rustdoc::missing_doc_code_examples`), the original lint level definition note is no longer for `rustdoc::all`, so it shows up again here.

So this makes sense now 👍

Reply:
I see, thanks for the explanations!